microsoft-edge/progressive-web-apps-chromium/whats-new/pwa.md

```diff
@@ -281,7 +281,7 @@ Microsoft Edge version 92 became the stable channel of Microsoft Edge on July 22
 
 #### Protocol handlers origin trial
 
-You can now register your PWA to handle specific protocols with the host operating system. The Windows trial for protocol handlers is now available. You can register your origin for the **Web App Protocol Handler** trial at the [origin trial signup page](https://developer.microsoft.com/microsoft-edge/origin-trials/web-app-protocol-handler-registration/registration).
+You can now register your PWA to handle specific protocols with the host operating system. The Windows trial for protocol handlers is now available. You can register your origin for the **Web App Protocol Handler** trial at the [origin trial signup page](https://developer.microsoft.com/microsoft-edge/origin-trials/web-app-protocol-handler-registration/registration).<!-- todo: update per https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/ -->
```

Comment:
3
@captainbrosset @amandabaker - todo: update per https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/

Reply:
Same as above.

Reply:
result:
```
You can now register your PWA to handle specific protocols with the host operating system.
Learn more about using protocol handlers with your PWA at 
[Handle protocols in Progressive Web Apps](../how-to/handle-protocols.md).

_2023 update: This feature is no longer in [Origin Trial](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/), 
so the following no longer applies:_  
The Windows trial for protocol handlers is now available. 
You can register your origin for the **Web App Protocol Handler** trial at the 
[origin trial signup page](https://developer.microsoft.com/microsoft-edge/origin-trials/web-app-protocol-handler-registration/registration).
```
and will add Note at top of article